extensions/linear/src/helpers/projects.ts

```diff
@@ -40,3 +43,20 @@ export const projectStatusText: Record<string, string> = {
   completed: "Completed",
   canceled: "Canceled",
 };
+
+export function getProjectIcon(project?: Pick<Project, "icon" | "color">) {
+  if (!project) {
+    return { source: { light: "light/no-project.svg", dark: "dark/no-project.svg" } };
+  }
+
+  const emojiRegex = new RegExp(/:[a-z0-9_+-]+:/, "g");
+
+  if (project.icon && emojiRegex.test(project.icon)) {
+    return emojis.get(project.icon);
```

Comment:
there are no colors when you have an emoji?

Reply:
No need to set a color as Raycast as the source of the image can be a single emoji (cf [Image.Source](https://developers.raycast.com/api-reference/user-interface/icons-and-images#image.source)) 🙂

Reply:
yeah I know, I was wondering if you could specify a color with an emoji on Linear. Turns out no but I was too lazy to check 😄

Reply:
Ah, my bad 😄